h2o-algos/src/main/java/hex/tree/gbm/GBMModel.java

```diff
@@ -15,6 +15,14 @@
     public double _col_sample_rate;
     public double _max_abs_leafnode_pred;
     public double _pred_noise_bandwidth;
+    // TODO these are only for debugging / meassuring effects of new histo task, should be removed
```

Comment:
i'd change the comment to say that these are the best heuristics we have (i.e., those shouldn't be removed, as they are needed)